LocalSettings.php

```diff
@@ -1809,7 +1809,7 @@
 		'extloadwiki' => true,
 		'grandtheftautowiki' => true,
 		'inazumaelevenwiki' => true,
-		'thebbwiki' => true,
+		'thebbwiki' => false,
```

Comment:
If you're settings something to false, but the default is false, just remove it.